index.html

```diff
@@ -674,12 +677,12 @@ <h3>Validate Claim</h3>
           is the domain of validation.
           <br><br>
           It's important to note that the bounds of authority for a given
-          issuer are specific claim types made by them and relied upon by
+          issuer are specific credential types made by them and relied upon by
```

Comment:
```suggestion
          issuer are specific claim types made by them and relied upon by
```


In this case, the granularity is the claim, not the credential.